src/lib/components/workflow/workflows-summary-configurable-table.svelte

```diff
@@ -18,8 +21,8 @@
   let customizationDrawerOpen: boolean = false;
 
   $: ({ namespace } = $page.params);
-  $: columns = $workflowTableColumns?.[namespace] ?? [];
-  $: empty = workflows.length === 0 || columns.length === 0;
+  $: columns = $workflowTableColumns?.[namespace] ?? getDefaultColumns();
+  $: empty = workflows.length === 0;
```

Comment:
Couple of edge cases still 😞 

First one is if there are workflows, but no columns are selected (we see the bulk actions)

![Screenshot 2023-07-10 at 12 40 53 PM](https://github.com/temporalio/ui/assets/15069288/1ffd26b3-d79d-44c8-b4e5-5e8e89287f96)

Second one is if there are only pinned columns

![Screenshot 2023-07-10 at 12 41 33 PM](https://github.com/temporalio/ui/assets/15069288/866a42b8-d1fd-41ff-b662-68f7098bfaf4)